packages/esm-generic-patient-widgets-app/src/resources/useObs.ts

```diff
@@ -18,35 +18,42 @@ export function useObs(patientUuid: string): UseObsResult {
       data.map((d) => d.concept).join(',') +
       '&_summary=data&_sort=-date' +
       `&_count=${pageSize}` +
-      urlEncounterTypes,
+      urlEncounterTypes +
+      '&_include=Observation:encounter',
```

Comment:
I'd prefer that we have this branching instead of integrated, that is, if I _don't_ enable showing encounters, I shouldn't have to process them at all.